kairos-deposit-contract/readme.md

```diff
@@ -94,7 +94,7 @@ For now, on Macos:
 
 or, on Ubuntu/Debian:
 
-`./build-ubuntu-feat-2.0.sh`
+`./build-ubuntu-feat-2.0.sh` 
```

Comment:
This is not serving the purpose adding a space to sign this specific commit. All the other commits are still unsigned. please read https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits and configure your git accordingly